code/style.css

```diff
@@ -1,7 +1,428 @@
+}
+
+.hero-slide p {
+  font-family: 'Dancing Script', cursive;
+  position: absolute;
+  top: 60%;
+  left: 50%;
+  transform: translate(-50%, -50%);
+  font-size: 16px;
+  color: #ffffff;
+  margin: 0px;
+}
+
+.contact-link {
+  position: absolute;
+  /* display: inline; */
+  top: 10px;
+  right: 10px;
+  font-size: 18px;
+  color: #ffffff;
+  margin: 10px;
+  text-decoration: none;
+}
+
+.contact-link:hover {
+  color: black;
+}
+
+
+.slider {
+  -moz-perspective: 1300px;
+  -ms-perspective: 1300px;
+  -webkit-perspective: 1300px;
+  perspective: 1300px;
+  display: inline-block;
+  text-align: left;
+  position: relative;
+  margin-bottom: 22px;
+}
+
+.slider>input {
+  display: none;
+}
+
+.slider>input:nth-of-type(3):checked~ul li:first-of-type {
+  margin-left: -200%;
+}
+
+.slider>input:nth-of-type(2):checked~ul li:first-of-type {
+  margin-left: -100%;
+}
+
+.slider>input:nth-of-type(1):checked~ul li:first-of-type {
+  margin-left: 0%;
+}
+
+.slider>ul {
+  position: relative;
+  width: auto;
+
+  z-index: 1;
+  font-size: 0;
+  line-height: 0;
+  margin: 0 auto;
+  padding: 0;
+  overflow: hidden;
+  white-space: nowrap;
+  -moz-box-sizing: border-box;
+  -webkit-box-sizing: border-box;
+  box-sizing: border-box;
+}
+
+.slider>ul>li {
+  position: relative;
+  display: inline-block;
+  width: 100%;
+  height: 100%;
+  overflow: hidden;
+  font-size: 15px;
+}
+
+.slider>ul>li video {
+  width: 100%;
+}
+
+.slider>.dots {
+  position: absolute;
+  bottom: 10px;
+  z-index: 10;
 
-.active + .answer {
-  /* some styling here */
+  justify-content: center;
+  display: flex;
+  width: 100%;
+  /* height: 100%;
+  top: 90%;
+  left: 0; */
 }
+
+.slider>.dots label {
+  position: relative;
+  display: inline-block;
+  cursor: pointer;
+  border-radius: 50%;
+  margin: 0 4px;
+  padding: 5px;
+  background: white;
+}
+
+.slider>.dots label:after {
+  content: "";
+  position: absolute;
+  left: 50%;
+  top: 50%;
+  margin-left: -6px;
+  margin-top: -6px;
+  background: pink;
+  border-radius: 50%;
+  padding: 5px;
+  opacity: 0;
+}
+
+.slider > .dots label:hover:after {
+  opacity: 1;
+}
+
+
+/* ------SIGN-UP FORM STYLING ----- */
+form {
+  font-family: 'Open Sans Condensed', sans-serif;
+  margin: 10px;
+  font-size: 18px;
+  display: flex;
+  flex-direction: column;
+}
+
+.form-input {
+  /* display: inline-block; */
+  width: 85%;
+  padding: 10px;
+  margin: 5px 20px;
+  font-size: 14px;
+}
+
+.text-title {
+  font-family: 'Dancing Script', cursive;
+  font-size: 30px;
+}
+
+.message {
+  padding-bottom: 60px;
+}
+
+.radio-age {
+  display: flex;
+  /* flex-direction: row;
+  width: 100%; */
+  margin-left: 20px;
+}
+
+.sign-up {
+  width: 100%;
+  max-width: 700px;
+  min-width: 280px;
+  margin: auto;
+}
+
+.radio-button {
+  display: block;
+  /* position: relative; */
+  margin-right: 20px;
+  font-size: 16px;
+  user-select: none;
+
+}
+
+.checkbox {
+  margin-left: 20px;
+}
+
+.form-text {
+  padding: 0 20px;
+}
+
+fieldset {
```

Comment:
What does it mean that is "user agent stylesheet". You don't have to answer here, but it might be good to comment in the code if it something you linked somehow? (It would help the reviewer :))